packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -83,6 +86,43 @@ describe('snippet-bot', () => {
           },
         ],
       });
+      const testSnippets = new Map<string, Snippet>();
+      const languageMap = new Map<string, SnippetLanguage>();
+      languageMap.set('PYTHON', {
+        status: 'IMPLEMENTED',
+        current_locations: [
+          {
+            repository_path: 'tmatsuo/repo-automation-bots',
+            filename: 'test.py',
+            commit: 'xxx',
+            branch: 'master',
+          },
+        ],
+      });
+      testSnippets.set('datastore_incomplete_key', {
+        title: '',
+        description: '',
+        languages: languageMap,
+      });
+      const languageMap2 = new Map<string, SnippetLanguage>();
+      languageMap2.set('PYTHON', {
+        status: 'CONFLICT',
+        current_locations: [
+          {
+            repository_path: 'tmatsuo/repo-automation-bots',
+            filename: 'test.py',
+            commit: 'xxx',
+            branch: 'master',
+          },
+        ],
+      });
+      testSnippets.set('datastore_named_key', {
+        title: '',
+        description: '',
+        languages: languageMap2,
+      });
+      getSnippetsStub = sandbox.stub(snippetsModule, 'getSnippets');
```

Comment:
mind adding an assertion that specifically describes the new behavior, e.g.,

"should add warning when this condition is met".

Reply:
Sure, done